docs/fast_data/architecture.md

```diff
@@ -81,8 +81,8 @@ Fast Data architecture is rather streamlined, with just a couple of pivoting poi
 1. The CDC emits an event stating that some data in the SoR has changed;
 2. The RTU performs the normalization of the messages received by the CDC to select the ones of interest and make them adhere to a standard of interest, and then stores the Projections on MongoDB;
    1. The RTU emits a `Projection Update` event, if it is configured to do so;
-3. The RTU or the SVT (depending on which one you chose for your architecture) compute and emit a `Projection Change` event, saving it either on Kafka or on MongoDB
-4. The SVC reads the `Projection Change` message, either polling MongoDB or reacting to the Kafka message. Then, it aggregates the Single View using the new data, and stores it to MongoDB;
+3. The RTU or the SVTG (depending on which one you chose for your architecture) compute and emit a `Projection Change` or `sv-trigger` event, saving it either on MongoDB or Kafka
```

Comment:
@ianFar96 let's add to the glossary page that the Real Time Updater is called RTU, Single View Creator is called SVC and so on. So that it is written centralized somewhere that we use these abbreviations

Reply:
I added this extra col called alias
![image](https://github.com/mia-platform/documentation/assets/30315670/0bf3ba90-5a77-4218-a1a0-65522c8b5253)

Reply:
very nice!!